thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -211,6 +213,72 @@ public Response computeSeverity(@NotNull @QueryParam("collection") String collec
   }
 
   /**
+   * The wrapper endpoint to do first time replay, tuning and send out notification to user
+   * @param id new anomaly function id
+   * @param startTimeIso start time of replay
+   * @param endTimeIso end time of replay
+   * @param isForceBackfill whether force back fill or not
+   * @param speedup whether use speedUp or not
+   * @param userDefinedPattern tuning paramter, user defined pattern can be "UP", "DOWN", or "UP&DOWN"
+   * @param sensitivity sensitivity level for initial tuning
+   * @param fromAddr email notification from address
+   * @param toAddr email notification to address
+   * @param subject subject of email
+   * @param teHost thirdeye host
+   * @param smtpHost smtp host
+   * @param smtpPort smtp port
+   * @param phantomJsPath phantomJSpath
+   * @return
+   */
+  @POST
+  @Path("/{id}/notifyreplaytuning")
+  public Response triggerReplayTuningAndNotification(@PathParam("id") @NotNull final long id,
+      @QueryParam("start") @NotNull String startTimeIso,
+      @QueryParam("end") @NotNull String endTimeIso,
+      @QueryParam("force") @DefaultValue("false") String isForceBackfill,
+      @QueryParam("speedup") @DefaultValue("false") final Boolean speedup,
+      @QueryParam("userDefinedPattern") @DefaultValue("UP") String userDefinedPattern,
+      @QueryParam("sensitivity") @DefaultValue("MEDIAN") final String sensitivity,
+      @QueryParam("from") String fromAddr,
+      @QueryParam("to") String toAddr, @QueryParam("subject") String subject,
```

Comment:
Was giving the freedom to let user customize the subject of replay email. Modified to use "Replay results for {alert name} is ready for review!" 
As for fromAddr, usually uses thirdeye_dev@ for testing, currently still leave it as a parameter and let frontend input the fromAddr.